keptn/post_install.md

```diff
@@ -18,18 +18,16 @@
 
 
 1. Connect the CLI to the Keptn installation
-  ```
-  API_PORT=$(kubectl get svc api-gateway-nginx -n keptn -o jsonpath='{.spec.ports[?(@.name=="http")].nodePort}')
-  EXTERNAL_NODE_IP=$(kubectl get nodes -o jsonpath='{ $.items[0].status.addresses[?(@.type=="ExternalIP")].address }')
-  KEPTN_ENDPOINT=http://${EXTERNAL_NODE_IP}:${API_PORT}/api
+  ```  
+  KEPTN_ENDPOINT=http://$(kubectl -n keptn get service api-gateway-nginx -o jsonpath='{$.status.loadBalancer.ingress[0].ip}')/api
   KEPTN_API_TOKEN=$(kubectl get secret keptn-api-token -n keptn -ojsonpath={.data.keptn-api-token} | base64 --decode)
 
   keptn auth --endpoint=$KEPTN_ENDPOINT --api-token=$KEPTN_API_TOKEN
   ```
 
 1. Access the bridge via:
   ```
-  echo http://${EXTERNAL_NODE_IP}:${API_PORT}/bridge
+  echo http://$(kubectl -n keptn get service api-gateway-nginx -o jsonpath='{$.status.loadBalancer.ingress[0].ip}')/bridge
```

Comment:
Please merge the instruction for accessing the bridge.
The following command will give you the endpoint + credentials.
```
keptn configure bridge --output
```